bob/src/cluster/operations.rs

```diff
@@ -43,14 +45,15 @@ pub(crate) async fn finish_at_least_handles<TErr: Debug>(
     errors
 }
 
-async fn call_at_least<TOp, TErr: Debug>(
+async fn call_at_least<TOp, TRes, TErr: Debug>(
     target_nodes: impl Iterator<Item = TOp>,
     at_least: usize,
-    f: impl Fn(TOp) -> JoinHandle<Result<NodeOutput<()>, NodeOutput<TErr>>>,
-) -> (FuturesUnordered<JoinHandle<Result<NodeOutput<()>, NodeOutput<TErr>>>>, Vec<NodeOutput<TErr>>) {
+    f: impl Fn(TOp) -> JoinHandle<Result<NodeOutput<TRes>, NodeOutput<TErr>>>,
+    get_affected_count: impl Fn(TRes) -> usize
```

Comment:
`call_at_least` should not know anything about replicas count. It is a general function. It is better to propagate the number of affected replicas through the vector with successful results